youtube_dl/extractor/mtv.py

```diff
@@ -201,7 +201,7 @@ def _get_videos_info_from_url(self, url, video_id):
             [self._get_video_info(item) for item in idoc.findall('.//item')],
             playlist_title=title, playlist_description=description)
 
-    def _extract_mgid(self, webpage):
+    def _extract_mgid(self, webpage, fatal=True):
```

Comment:
That's funny, because I initially wrote this fix using `default`, then I found that `fatal` existed, and that seemed like a cleaner way to write it.